forte_wrapper/spacy/spacy_processors.py

```diff
@@ -115,6 +137,40 @@ def _process_ner(self, result, input_pack):
                                    item.end_char)
             entity.ner_type = item.label_
 
+    def _process_umls_entity_linking(self, result, input_pack: DataPack):
+        """
+        Do UMLS medical entity linking with scispacy, and store medical entity
+        mentions and UMLS concepts.
+        Args:
+            result: SpaCy results
+            input_pack: Input datapack
+        Returns:
+        """
+        medical_entities = result.ents
```

Comment:
check whether all models support ner and linking, leave todo notes and issues if not